infra/main.tf

```diff
@@ -13,22 +13,23 @@ resource "ovh_cloud_project_kube_nodepool" "node_pool" {
   service_name  = var.service_name
   kube_id       = each.value.id
   name          = "${each.value.name}-nodepool"
-  flavor_name   = "d2-8"
-  desired_nodes = 3
-  max_nodes     = 3
-  min_nodes     = 3
+  flavor_name   = var.ovh_k8s_flavor
+  autoscale     = var.ovh_k8s_autoscale
+  desired_nodes = var.ovh_k8s_disired_nodes
+  max_nodes     = var.ovh_k8s_max_nodes
+  min_nodes     = var.ovh_k8s_min_nodes
 }
 
-resource "local_file" "kubeconfig" {
+/*resource "local_file" "kubeconfig" {
   count    = var.number_of_clusters
   content  = ovh_cloud_project_kube.my_kube_cluster[count.index].kubeconfig
-  filename = "polytech-${count.index + 1}.yml"
+  filename = "${var.cluster_name}-${count.index}.yml"
 
   // Prometheus operator deployment
   provisioner "local-exec" {
     command = <<EOT
               helm install prometheus-community/kube-prometheus-stack \
-                            --kubeconfig polytech-${count.index + 1}.yml \
+                            --kubeconfig polytech-${count.index}.yml \
```

Comment:
polytech reference still exists here?

Reply:
${var.cluster_name}-${count.index}.yml

?

Reply:
true event it"s in comment, I will have a bad surprise if i uncomment it, thanks 😉.

fix by https://github.com/philippart-s/labs-on-ovh/pull/4/commits/fd22db77ddd562670f970baa610ead4a2c698d91